command/agent/command.go

```diff
@@ -301,6 +307,67 @@ func (c *Command) readConfig() *Config {
 	return config
 }
 
+// verifyUniqueListeners checks to see if an address was used more than once in
+// the config
+func (config *Config) verifyUniqueListeners() (bool, error) {
+	type key struct {
+		host string
+		port int
+	}
+	const numUniqueAddrs = 7
+	m := make(map[key]string, numUniqueAddrs)
+
+	testFunc := func(k key, descr string) (bool, error) {
```

Comment:
The `bool` here is redundant with the `error` - just return a `nil` error I think.